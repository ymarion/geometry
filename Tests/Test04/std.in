# Creating a Rectangle
R R1 0 0 20 20
# Should show R1
LIST


