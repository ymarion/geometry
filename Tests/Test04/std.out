# Creation
OK
# LIST
R R1 0 0 20 20
